quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -111,6 +111,46 @@ public RawKeycloakDistribution(boolean debug, boolean manualStop, boolean enable
         this.distPath = prepareDistribution();
     }
 
+    public CLIResult kcadm(List<String> arguments) throws IOException {
```

Comment:
Just a nitpick. We could use varargs gere to avoid the need to create Lists in the tests that call this.

Reply:
It's just mirroring the other run call. Do you want vararg signatures for both?

Reply:
Right but there's also this:
https://github.com/keycloak/keycloak/blob/7f048300fe52a55ea99527538ad9d57b4a543ee4/quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/KeycloakDistribution.java#L14-L16
Maybe we could do something like it as well?

Reply:
Ok, added